.github/workflows/balena-ci.yml

```
@@ -5,19 +5,22 @@ on:
       - master
 
 jobs:
-  build:
+  balena-cloud-build:
     runs-on: ubuntu-20.04
     strategy:
+      fail-fast: true
       matrix:
-        fail-fast: true
         fleets: [testbot-rig, testbot-rig-beta, testbot-personal]
     steps:
       - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v2
        with:
          submodules: true
```
We need the contracts submodule.